src/core/ledger/applyDistributions.js

```diff
@@ -47,7 +48,9 @@ export function applyDistributions(
   const credIntervals = credGraph.intervals();
   const distributionIntervals = _chooseDistributionIntervals(
     credIntervals,
-    ledger.lastDistributionTimestamp(),
+    // Distributions after -Infinity are all the current
+    // distributions
```

Comment:
We tend to be conservative about non-documentation, inline comments. Often they add more confusion than clarity, and not leaning on comments forces us to write code that is readable and understandable on its own. I'd remove this comment.

Reply:
That makes sense, I'm also on the side of preferring readable code than overly hand-holdy comment lol, I applied these changes